ton_client/client/src/contracts/deploy.rs

```diff
@@ -124,6 +153,80 @@ pub(crate) fn encode_message(_context: &mut ClientContext, params: ParamsOfDeplo
     })
 }
 
+pub(crate) fn get_code_from_image(_context: &mut ClientContext, params: ParamsOfGetCodeFromImage) -> ApiResult<ResultOfGetCodeFromImage> {
+    debug!("-> contracts.deploy.image.code()");
+
+    let bytes = base64::decode(&params.imageBase64)
+        .map_err(|err| ApiError::contracts_deploy_invalid_image(err))?;
+    let mut reader = Cursor::new(bytes);
+    let image = ContractImage::from_state_init(&mut reader)
+        .map_err(|err| ApiError::contracts_deploy_image_creation_failed(err))?;
+
+    debug!("<-");
+    Ok(ResultOfGetCodeFromImage {
+        codeBase64: base64::encode(&image.get_serialized_code()
+            .map_err(|err| ApiError::contracts_deploy_image_creation_failed(err))?),
+    })
+}
+
+pub(crate) fn get_deploy_data(_context: &mut ClientContext, params: ParamsOfGetDeployData) -> ApiResult<ResultOfGetDeployData> {
```

Comment:
A lot of work in one function. Can it be splitted to 2 parts? update image data and return data from image.

Reply:
I don't think it is needed. It's splitting just for splitting